google/cloud/pubsub/internal/tracing_helpers.cc

```diff
@@ -35,6 +37,43 @@ opentelemetry::trace::StartSpanOptions RootStartSpanOptions() {
   return options;
 }
 
+std::vector<
+    std::pair<opentelemetry::trace::SpanContext,
+              std::vector<std::pair<opentelemetry::nostd::string_view,
+                                    opentelemetry::common::AttributeValue>>>>
+CreateLinks(opentelemetry::trace::SpanContext const& span_context) {
+  using Attributes =
+      std::vector<std::pair<opentelemetry::nostd::string_view,
+                            opentelemetry::common::AttributeValue>>;
+  using Links =
+      std::vector<std::pair<opentelemetry::trace::SpanContext, Attributes>>;
+#if OPENTELEMETRY_ABI_VERSION_NO >= 2
+  if (span_context.IsSampled() && span_context.IsValid()) {
+    return Links{{span_context, Attributes{}}};
+  }
+#endif
+  (void)span_context;
+  return Links{};
```

Comment:
I think you can cut the number of repeated aliases:

```suggestion
#if OPENTELEMETRY_ABI_VERSION_NO >= 2
  if (span_context.IsSampled() && span_context.IsValid()) {
    return {{span_context, TracingAttributes{}}};
  }
#endif
  (void)span_context;
  return {};
```